docs/LOCAL_DEPLOYMENT.md

```diff
@@ -0,0 +1,67 @@
+
+Local Deployment
+================
+### Overview
+This document and other documents, referenced here, contain steps to git clone the latest repository, deploying the microservices locally and also a walk through the general repository and microservice structure.
+&nbsp; 
+### Steps to clone the restructured repository
+#### If you have an existing cloned repository
+1) Use recurssive clone to clone the repository and the sub-modules.
```

Comment:
typo: recursive

Reply:
Thanks for correcting, completed most of them